plugins/org.eclipse.epsilon.eol.engine/src/org/eclipse/epsilon/eol/ImportManager.java

```diff
@@ -0,0 +1,66 @@
+package org.eclipse.epsilon.eol;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.eclipse.epsilon.common.module.IModule;
+import org.eclipse.epsilon.common.util.UriUtil;
+import org.eclipse.epsilon.eol.dom.Import;
+
+public class ImportManager implements IImportManager {
+	
+	protected Map<URI, IModule> cache = new HashMap<>();
+
+	@Override
+	public void loadModuleForImport(Import import_, Class<? extends IModule> moduleImplClass, URI baseURI) {
+		
+		String importPath = import_.getPath();
+		
+		URI importUri;
+		try {
+			importUri = UriUtil.resolve(importPath, baseURI).normalize();
+		}
+		catch (URISyntaxException ex) {
+			// Return with import not loaded.
+			return;
```

Comment:
Should we not throw a runtime exception at least in this case? In fact, I think this loadModuleForImport method should be throwing some sort of checked exception.